### ISSUE TITLE
loss during training

### ISSUE THREAD

**Issue by Amberlin1970** [kobiso/CBAM-tensorflow-slim#3]
Hello！I have a question: should I train my model until the loss becomes the end learning rate(0.0001)? If I stop training my model when the loss is 0.027, then my model will perform badly?

**Comment by cherry-and-leaves** [comment]
Train and validate each epoch, and you know when the model is best
